docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/CrossValidationSplit.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using static Microsoft.ML.DataOperationsCatalog;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    /// <summary>
+    /// Sample class showing how to use CrossValidationSplit.
+    /// </summary>
+    public static class CrossValidationSplit
+    {
+        public static void Example()
+        {
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline.
+            var mlContext = new MLContext();
+
+            // Generate some data points.
+            var examples = GenerateRandomDataPoints(10);
+
+            // Convert the examples list to an IDataView object, which is consumable by ML.NET API.
+            var dataview = mlContext.Data.LoadFromEnumerable(examples);
+            // Cross validation splits your data randomly into set of chunks, and set test set in each fold to be certain chunk of data
```

Comment:
Maybe

" // Cross validation splits your data randomly into set of chunks, and set test set in each fold to be certain chunk of data and training data set to be rest of the data."

to "Cross validation splits your data randomly into set of "folds", and creates groups of Train and Test sets, where for each group, one fold is the Test and the rest of the folds the Train."

"